playbooks/common/openshift-cluster/upgrades/v3_1_minor/pre.yml

```diff
@@ -41,7 +41,7 @@
       g_new_service_name: "{{ 'origin' if deployment_type =='origin' else 'atomic-openshift' }}"
 
   - name: Determine available versions
-    script: ../files/versions.sh {{ g_new_service_name }} openshift
+    script: ../files/versions.sh {{ g_new_service_name }}
```

Comment:
@sdodson, @detiber, this is another one of those things that I have no idea how anyone's 3.1.1 upgrade actually worked.  This needs to be fixed asap.  In the minor upgrade we only need to know the latest version of the atomic-openshift package.